yrs/src/block.rs

```diff
@@ -419,7 +459,7 @@ pub(crate) struct Item {
     pub parent_sub: Option<String>, //TODO: Rc since it's already used in Branch.map component
 
     /// Bit flag field which contains information about specifics of this item.
-    pub info: Cell<u8>,
+    pub info: u8,
```

Comment:
Since block store now accesses `Block`s via `UnsafeCell`, we no longer need cells here. We can just obtain mutable block reference at any point in time.